sbin/travis.sh

```diff
@@ -79,6 +113,41 @@ RunTests() {
             sh -e /etc/init.d/xvfb start
         fi
     fi
+
+	# Install dependencies through Package Control
+	if [ -n $PCDEPS ]; then
+		STIP="${STP%/*}/Installed Packages"
+		if [ ! -d "$STIP" ]; then
+			echo creating sublime installed package directory
+			mkdir -p "$STIP"
+		fi
+	
+		# Install PackageControl
+		PC_URL="https://packagecontrol.io/Package Control.sublime-package"
+		PC_PKG="${PC_URL##*/}"
+		curl "$PC_URL" -o "$STIP/$PC_PKG"
+
+		# Cycle ST to complete installation
+		finished="! awk '/in_process_packages/,/]/' \
+			'$STP/User/Package Control.sublime-settings' | tail -n +2 | grep -q \\\""
+
+		echo Installing Package Control...
+		CycleUntil "[ -f '$STP/User/Package Control.sublime-settings' ] && $finished"
+
+		echo Opening Sublime until Package Control installs...
+		CycleUntil "[ -d '$STP/bz2' ] && $finished"
```

Comment:
It seems that this has already installed your Package PC dependencies. It triggers the [automatic upgrader](https://github.com/wbond/package_control/blob/5643f4fa9a335a4591482a8ecdf6e0d7413fbaa1/package_control/automatic_upgrader.py) and finds the [missing dependencies](https://github.com/wbond/package_control/blob/5643f4fa9a335a4591482a8ecdf6e0d7413fbaa1/package_control/package_manager.py#L694) of all packages. So there is no need of `--command install_local_dependency`?